stubs/backend/github/github-stubs.yaml

```diff
@@ -52,4 +52,16 @@
     status: 200
     headers:
       content-type: application/json
-    file: ./backend/github/jsons/github.user.repos.pull.request.commits.json
+    file: ./backend/github/jsons/github.repos.pull.request.commits.json
+
+# GitHub Commit Data
+# Example: repos/xxx/xxx/commits/commit-sha
+- request:
+    method: GET
+    url: /repos/([\w.-]+\/[\w.-]+)/commits/(\b[0-9a-z]+\b)
+
+  response:
+    status: 200
+    headers:
+      content-type: application/json
+    file: ./backend/github/jsons/github.repos.commit-sha.json
```

Comment:
Should be ./backend/github/jsons/github.repos.fb1b445a113ef2d1ed465732a5dd02c2e07ca1c4.json
Incase we will add more commit as mock data.